views/forgotPassword.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<!--[if lt IE 9]>
+	<script src="dist/html5shiv.js"></script>
+	<![endif]-->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no" />	
+	<title>Meaningful - Forgot Password</title>
+	<link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,700|Merriweather:400,300italic,300,700,900' rel='stylesheet' type='text/css'>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+
+	<link rel="stylesheet" type="text/css" href="/stylesheets/main.css">
+	<link rel="stylesheet" type="text/css" href="/stylesheets/login.css">
+</head>
+<body>
+	<div class="wrapper">
+		<a href="/login">< back to login</a>
+		<div class="centered">
+			<h2>I Forgot My Password!</h2>
```

Comment:
Do we want to have every word capitalized here? I can see that it could be good to do since it's a header, but as a user I think it might be more intuitive to see it written as a sentence: `I forgot my password!` What do you think?